### ISSUE TITLE
Browse by keyword and search have different result views

### ISSUE THREAD

**Issue by goliatone** [npm/newww#897]
Search by a term, **task**:
 - url: https://www.npmjs.com/search?q=task
The results show this information per entry:
```
grunt
The JavaScript Task Runner
version 0.4.5
295450 downloads in the last week
```

Search by keyword, **task**:
- url: https://www.npmjs.com/browse/keyword/task
```
grunt
The JavaScript Task Runner
0.4.5 published a month ago by cowboy
```

I expect to see the same information about the package in both views. 

Of the two, I find the one showing number of downloads more useful. 

Ideally thou, it would be a merge of the two, where you can see both the author and the number of downloads.

**Comment by KenanY** [comment]
Duplicate of https://github.com/npm/newww/issues/599?

**Comment by KenanY** [comment]
Oh I should have read your full post before I posted :(

**Comment by goliatone** [comment]
Yes, I believe is a different issue. I understand the results shown will be different, however, the information for each result should be consistent.
